src/components/RequiredAsterisk.vue

```diff
@@ -21,18 +21,18 @@ export default {
 </template>
 
 <style lang="scss" scoped>
-@import '@wmde/wikit-tokens/variables';
+@import '@wikimedia/codex-design-tokens/theme-wikimedia-ui';
 
 .wbl-snl-required-asterisk {
-	font-size: $font-size-xxlarge;
+	font-size: $font-size-xx-large;
```

Comment:
This changes the font size from `1.5em` to `1.5rem`, which actually makes a difference (at least in legacy Vector) – I don’t think this is expected by T369509, which says that “that same token […] matches Codex already”?

Reply:
Now `1.25rem`, which comes out to `20px` rather than `21px` (smaller change).